drivers/ethernet/oa_tc6.c

```diff
@@ -9,6 +9,14 @@
 #include <zephyr/logging/log.h>
 LOG_MODULE_REGISTER(oa_tc6, CONFIG_ETHERNET_LOG_LEVEL);
 
+/*
+ * When IPv6 support enabled - the minimal size of network buffer
+ * shall be at least 128 bytes (i.e. default value).
+ */
+#if (CONFIG_NET_IPV6 == 1) && (CONFIG_NET_BUF_DATA_SIZE < 128)
```

Comment:
More readable would be to use the "defined" here
```
#if defined(CONFIG_NET_IPV6) && (CONFIG_NET_BUF_DATA_SIZE < 128)
```

Reply:
@jukkar - unfortunately not - in many *.conf extensions you just have `CONFIG_NET_IPV6=n` which would return true with `#if defined()`

Reply:
Our code base is full of `defined(...)` constructs and everything works just fine if you do disable that config option by setting it to `n`.  The reason is that the final `.config` file does not have any options which are disabled. So it is not possible to have `CONFIG_NET_IPV6=n` set in final `.config` file even if you set it in your own *.conf file.

Reply:
I would like to get this changed to use `defined()` instead of this, the config symbols are checked like this in other part of the code base and it is more readable that way.

Reply:
Ok, I will change it.